### ISSUE TITLE
Get min_q and max_q at current power

### ISSUE THREAD

**Issue by vincentbarbesant** [powsybl/pypowsybl#245]
* **Do you want to request a *feature* or report a *bug*?**
A feature

* **What is the current behavior?**
In the generators dataframe, `min_q` and `max_q` columns are filled with `NaNs` :

```
         name energy_source   target_p  min_p   max_p  min_q  max_q    target_v  target_q  voltage_regulator_on          p          q           i voltage_level_id     bus_id  connected        genreCvg
id                                                                                                                                                                                                      
.AGUAT 1              OTHER  43.000000    0.0  200.00    NaN    NaN  232.933334 -0.000000                  True -43.000000  75.072884  214.438040          .AGUA 6  .AGUA 6_0       True          FICTIF
.AGUAT 2              OTHER  42.000000    0.0  200.00    NaN    NaN  232.933334 -0.000000                  True -42.000000  75.072884  213.216969          .AGUA 6  .AGUA 6_0       True          FICTIF
.AMORT 1              OTHER   0.000000    0.0  250.00    NaN    NaN  418.323700 -0.000000                  True  -0.000000   1.536397    2.120461          .AMOR 7  .AMOR 7_0       True          FICTIF
.AMORT 2              OTHER   0.000000    0.0  250.00    NaN    NaN  418.323700 -0.000000                  True  -0.000000   1.536397    2.120461          .AMOR 7  .AMOR 7_0       True          FICTIF
.AMORT 3              OTHER   0.000000    0.0  250.00    NaN    NaN  418.323700  0.490841                  True  -0.000000   1.536397    2.120461          .AMOR 7  .AMOR 7_0       True          FICTIF
...       ...           ...        ...    ...     ...    ...    ...         ...       ...                   ...        ...        ...         ...              ...        ...        ...             ...
VOVESIN3               WIND   0.180462    0.0   16.50    NaN    NaN   93.666534  0.000000                 False  -0.180462  -0.000000    1.119832          VOVESP4  VOVESP4_0       True          EOLIEN
WALDIIN1              SOLAR   0.000000    0.0    2.45    NaN    NaN   65.146873  0.000000                 False  -0.000000        NaN         NaN          WALDIP3                 False  PHOTOVOLTAIQUE
WALDIIN3              SOLAR   0.000000    0.0    2.45    NaN    NaN   65.146873  0.000000                 False  -0.000000        NaN         NaN          WALDIP3                 False  PHOTOVOLTAIQUE
YQUELIN1              SOLAR   0.000000    0.0    2.00    NaN    NaN   94.075058  0.000000                 False  -0.000000        NaN         NaN          YQUELP4                 False  PHOTOVOLTAIQUE
YZEURIN1              SOLAR   0.000000    0.0   10.20    NaN    NaN   65.871506  0.000000                 False  -0.000000        NaN         NaN          YZEURP3                 False  PHOTOVOLTAIQUE

```

* **What is the expected behavior?**
`min_q` and `max_q` columns are filled with their respective values at current power.

* **What is the motivation / use case for changing the behavior?**
I have to compute these limits myself, even though this function exists in powsybl and could be used at the dataframe creation

* **Please tell us about your environment:**
  - PowSyBl Version: 0.10.0
  - OS Version: Windows 10

**Comment by sylvlecl** [comment]
@geofjamg : 
I would propose to :
 - fill `min_q` and `max_q` with the value for current active power
 - add a column for the reactive limits kind (CURVE or MIN_MAX)
 - it would make sense to add a table for MIN_MAX kind with min_q and max_q, but it will be redundant with the existing columns, so I propose not to add it

Does it sound OK?

**Comment by geofjamg** [comment]
The problem with this design is that we are consusing the structural min/max and the current min/max at current active flow, but indeed in case of min/max limit it is the same.
So why not, but we have to be clear in the documentation about what is a fixed value and in which case and what might change depending on the active power.

**Comment by sylvlecl** [comment]
On this issue, I would propose to add 4 optional columns: `min_q_at_target_p`/`max_q_at_target_p`, `min_q_at_p`/`max_q_at_p`, in order to clarify the ambiguity.

Then `min_q` and `max_q` would stay `NaN` as today for CURVE limits.
Also, for ease of use, I would propose to make the `min_q` and `max_q` updatable when the generator already has MIN_MAX limits.

@CBiasuzzi  @geofjamg : does it sound OK ?

**Comment by geofjamg** [comment]
Ok for me

**Comment by CBiasuzzi** [comment]
Ok. I guess it would be useful to keep the "reactive limits kind" as a default column.
Should we make it optional like the 4 new columns, instead?

**Comment by sylvlecl** [comment]
Yes let's keep it as a default column!

**Comment by sylvlecl** [comment]
Implemented in #267
